src/overlay/OverlayManagerImpl.cpp

```diff
@@ -515,11 +671,15 @@ std::vector<Peer::pointer>
 OverlayManagerImpl::getRandomAuthenticatedPeers()
 {
     auto goodPeers = std::vector<Peer::pointer>{};
-    std::transform(std::begin(mAuthenticatedPeers),
-                   std::end(mAuthenticatedPeers), std::back_inserter(goodPeers),
-                   [](std::pair<NodeID, Peer::pointer> const& peer) {
-                       return peer.second;
-                   });
+    auto extractPeer = [](std::pair<NodeID, Peer::pointer> const& peer) {
+        return peer.second;
+    };
+    std::transform(std::begin(mInboundPeers.mAuthenticated),
+                   std::end(mInboundPeers.mAuthenticated),
+                   std::back_inserter(goodPeers), extractPeer);
+    std::transform(std::begin(mOutboundPeers.mAuthenticated),
+                   std::end(mOutboundPeers.mAuthenticated),
+                   std::back_inserter(goodPeers), extractPeer);
     std::random_shuffle(goodPeers.begin(), goodPeers.end());
```

Comment:
I see it was like this before, but this is probably a good opportunity to replace `std::random_shuffle` with `std::shuffle` here as well.